config/params/testnet_e2e_config.go

```diff
@@ -91,7 +89,7 @@ func E2EMainnetTestConfig() *BeaconChainConfig {
 	e2eConfig.TerminalTotalDifficulty = "616"
 
 	// Prysm constants.
-	e2eConfig.ConfigName = EndToEndName
+	e2eConfig.ConfigName = EndToEndMainnetName
```

Comment:
enforcing config name uniqueness turned up this little conflict, both e2e tests were using the same name constant.